setup.rst

```diff
@@ -59,10 +59,10 @@ Symfony application using Composer:
 .. code-block:: terminal
 
     # run this if you are building a traditional web application
-    $ composer create-project symfony/website-skeleton my_project_name
+    $ composer create-project symfony/website-skeleton:v4.4.99 my_project_name
```

Comment:
```suggestion
    $ composer create-project symfony/website-skeleton:"^4.4" my_project_name
```

Reply:
I did this change while merging. Thanks!